### PR TITLE
misc bugfix

### PR DESCRIPTION
**Pull request by lin755**

### CODE REVIEW

**Comment by gaozhangfei** [comment]
WARNING: From:/Signed-off-by: email name mismatch: 'From: taoqi 30043660 <taoqi10@huawei.com>' != 'Signed-off-by: taoqi <taoqi10@huawei.com>'

not standard name
Signed-off-by: taoqi 30043660 <taoqi10@huawei.com>

File: wd_sched.c [hunk 1]
```diff
@@ -480,7 +489,8 @@ static void wd_sched_map_cpus_to_dev(struct wd_sched_ctx *sched_ctx)
 	}
 }
 
-int wd_sched_rr_instance(const struct wd_sched *sched, struct sched_params *param)
+int wd_sched_rr_instance(const struct wd_sched *sched,
+	struct sched_params *param)
```
Comment (gaozhangfei) [inline wd_sched.c:493]: better to align

Comment (lin755) [reply wd_sched.c]: fixed

File: wd_mempool.c [hunk 2]
```diff
@@ -230,8 +230,12 @@ static struct bitmap *create_bitmap(int bits)
 
 static void destroy_bitmap(struct bitmap *bm)
 {
-	free(bm->map);
-	free(bm);
+	if (bm) {
+		if (bm->map)
+			free(bm->map);
```
[the remark this answers is not in the document]
Comment (lin755) [reply wd_mempool.c]: fixed

File: wd_sched.c [hunk 3]
```diff
@@ -177,6 +177,11 @@ static handle_t session_sched_init(handle_t h_sched_ctx, void *sched_param)
 	int node = numa_node_of_cpu(cpu);
 	struct sched_key *skey;
 
+	if (node < 0) {
+		WD_ERR("invalid: failed to get numa node!\n");
+		return (handle_t)(-WD_EINVAL);
```
[the remark this answers is not in the document]
Comment (lin755) [reply wd_sched.c:178]: it is used to avoid a static warning

**Comment by lin755** [comment]
> WARNING: From:/Signed-off-by: email name mismatch: 'From: taoqi 30043660 [taoqi10@huawei.com](mailto:taoqi10@huawei.com)' != 'Signed-off-by: taoqi [taoqi10@huawei.com](mailto:taoqi10@huawei.com)'
> 
> not standard name Signed-off-by: taoqi 30043660 [taoqi10@huawei.com](mailto:taoqi10@huawei.com)

fixed,thanks.

**Comment by gaozhangfei** [comment]
+1

**Review by Liulongfang — APPROVED** [review]
looks good to me

**Pull request merged**